### PR TITLE
refactor: move to Cloudflare workers

### PR DESCRIPTION
**Pull request by DTrombett**
**Changes this PR makes:**

It seems that replit will drop free hosting for discord bots. This means that it's time to move on!
I chose to try moving to Cloudflare Workers because they ~~have a free plan~~ are blazingly fast, have nearly no downtime and it is an opportunity to try the discord interactions via webhook.
This, sadly, has some disadvantages too because it makes difficult for us to implement the `/remind me` command and, mostly, the predictions logic.
However, I like tricky things, so this is where we're going! I'll be migrating all the commands, one by one, leaving the remind (probably, we'll need to drop it) and the predictions ones to the end.

PS: I'll probably ask @NotReallyEight to help me migrating some commands to conclude earlier 😁

### CODE REVIEW

**Comment by DTrombett** [comment]
Not really ready but I need a review anyway

**Comment by NotReallyEight** [comment]
holy moly

File: wrangler.toml
Comment (NotReallyEight) [inline wrangler.toml:1]: No trailing white space here

**Review by NotReallyEight — CHANGES_REQUESTED** [review]

File: wrangler.toml
Comment (DTrombett) [reply wrangler.toml:1]: What

Comment (NotReallyEight) [reply wrangler.toml:1]: image

**Review by NotReallyEight — APPROVED** [review]
LGTM.

**Pull request merged**